packages/core/lib/commands/opcode/run.js

```diff
@@ -29,15 +30,19 @@ module.exports = async function (options) {
   }
   const opcodes = CodeUtils.parseCode(bytecode, numInstructions);
 
-  const indexLength = (opcodes.length + "").length;
-
-  opcodes.forEach((opcode, index) => {
-    let strIndex = index + ":";
+  if (opcodes.length === 0) {
+    throw new TruffleError("List of instructions is empty!");
```

Comment:
Oh, hey, popping in here to say that this shouldn't error.  Running this on empty bytecode isn't an error.  (For isnstance, a contract may have empty bytecode if it's an interface or abstract.)  Truffle shouldn't error just because the user did something unusual.  You can have some sort of printout about empty bytecode and what this might indicate, but there's no reason for it to be an error.

Reply:
Would this solution work?

```suggestion
    console.log("List of instructions is empty!");
    return;
```

Reply:
@cds-amal, looks good to me! If everyone else agrees, I'll correct it like this

Reply:
How about this:
```
Contract has no bytecode. Please check to make sure it's not `abstract` or an `interface`
```

Reply:
I like @gnidan 's suggestion

Reply:
@gnidan 's suggestion looks good to me too.

Reply:
Do we need to tell them to check? I kind of like @cds-amal's as it just informs them. Although it seems like we should say, "No opcodes..." because they are requesting opcodes, right?

Reply:
Yeah, but if they're requesting opcodes for something that has no opcodes, it seems prudent to let them know about the disconnect there.